pkg/interface/weather/tile/tile.js

```diff
@@ -62,18 +46,22 @@ export default class WeatherTile extends Component {
 
   keyPress(e) {
     if (e.keyCode === 13) {
-      e.preventDefault();
-      this.manualLocationSubmit(e.target.value);
+
     }
   }
 
-  renderWrapper(child) {
+  renderWrapper(child,
+    weatherStyle = { gradient1: "white", gradient2: "white", text: "black" }
```

Comment:
weird that you aren't passing these in as separate arguments, as an object is *really* unstructured